src/routes/+page.svelte

```diff
@@ -15,15 +15,98 @@
 	onDestroy(unsubscribe);
 </script>
 
-<div class="hero min-h-screen bg-base-200">
+<div class="hero min-h-screen bg-base-100">
 	<div class="hero-content flex-col lg:flex-row-reverse">
 		<div class="text-center lg:text-left">
 			<h1 class="text-5xl font-bold">Start Your Pokemon Journey!</h1>
 		</div>
-		<div class="card shrink-0 w-full max-w-sm shadow-2xl bg-base-100">
+		<div class="card shrink-0 w-full max-w-sm shadow-2xl bg-neutral">
 			<SignUp {supabase} />
 		</div>
 	</div>
 </div>
 
-<p></p>
+<div class="welcome-banner">
+	<div class="font-bold max-w-xl mx-auto">
+		<h2>
+			Welcome to the Pokedex website, where you can delve into the vast world of Pokémon with ease.
+		</h2>
+	</div>
+
+	<div class="colOne"></div>
+	<div class="one font-bold">
+		<p>
+			We're more than just a Pokédex - we're a vibrant community of passionate Pokémon enthusiasts
+			like you! Explore our extensive database, discover the unique traits of each Pokémon species,
+			where you can connect with friends and share your Pokémon journey like never before.. okéAtlas
+			isn't just about information; it's about exploration and discovery. With an intuitive search
+			function and user-friendly interface, navigating the Pokédex is a breeze. Whether you're
+			searching by name, type, region, or even specific attributes like height or weight, PokéAtlas
+			makes it easy to find exactly what you're looking for.
+		</p>
+	</div>
+	<div class="colTwo"></div>
+	<div class="two font-bold">
+		<p>
+			dynamic community features. In addition to browsing the Pokédex, users can create their own
+			profiles, customize their collections, and connect with fellow trainers from around the world.
+			Whether you're sharing your latest catches, discussing strategy, or organizing trades and
+			battles, PokéAtlas provides a vibrant platform for connecting with other Pokémon fans.
+			<i>Ready to catch 'em all?</i>
+		</p>
+	</div>
+	<img src="/template-welcome-pokedex.png" alt="welcome background" height="100%" width="100%" />
+</div>
+
+<div class="Join Us flex justify-center">
```

Comment:
`Join Us` acts like two classes because of the space - use `join-us` instead. Do you actually need this class? If not, remove it.